src/main/java/org/venice/beachfront/bfapi/services/JobService.java

```diff
@@ -261,6 +282,41 @@ public void createJobError(Job job, String error) {
 					HttpStatus.INTERNAL_SERVER_ERROR);
 		}
 	}
+	
+	/**
+	 * Gets the detection data as a GPKG archive for a detection based on the Job ID
+	 * 
+	 * @param jobId
+	 *            The ID of the Job
+	 * @return The GPKG archive for the job
+	 */
+	public byte[] getDetectionGeoPackage(String jobId) throws UserException {
+		byte[] geoJsonData = this.getDetectionGeoJson(jobId);		
```

Comment:
Just a thought... in the future, we probably want to either 1) allow DB access to the Geopackage converter or 2) Move those Java components into BF-API directly. It seems silly that we're moving large GeoJSON from api->converter->api.  That is a lot of overhead.

Reply:
The GeoPackage converter can in theory fetch files directly from Piazza, so it would not be a stretch to add BF DB access as well. *However*, I agree that it probably makes more sense to have the conversion happen directly within the API. The only reason it was external before was due to the lack of good GPKG libraries in Python, and their existence in Java.

The way it's coded right now is probably fine for now (since we're re-implementing API 1.0 functionality), but an improvement in the near term could definitely be to eliminate the converter microservice entirely.